tests/Microsoft.Identity.Test.Unit/pop/PopAuthenticationSchemeTests.cs

```diff
@@ -140,7 +139,7 @@ public async Task ValidateKeyExpirationAsync()
                 var initialToken = result.AccessToken;
 
                 //Advance time 7 hours. Should still be the same key and token
-                testClock.TestTime +=TimeSpan.FromHours(7);
+                testClock.UtcNow +=TimeSpan.FromHours(7);
```

Comment:
The name Utsnow on the test clock seems a bit misleading. Seems like it should always be the current time but it changes as modifications are made to it. Would be better to name it UtcCurrent or UtcTestTime.